src/clims/migrations/0022_workbatch_input_containers.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.13 on 2020-12-14 14:13
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('clims', '0021_auto_20201110_0806'),
+    ]
+
+    operations = [
```

Comment:
This change wasn't directly changed to this PR, but I'm following up on this in the next PR, when I focus on the work batch itself.